### PR TITLE
Fix returning a string of json instead of just json

### PR DESCRIPTION
**Pull request by cwood**
This fixes a issue with go-sentry-api and it not being able to Unmarshal this error response.

### CODE REVIEW

File: src/sentry/api/endpoints/project_group_index.py [hunk 1]
```diff
@@ -883,7 +883,7 @@ def delete(self, request, project):
         else:
             # missing any kind of filter
             return Response(
-                '{"detail": "You must specify a list of IDs for this operation"}', status=400
```
Comment (cwood) [inline src/sentry/api/endpoints/project_group_index.py]: The go-sentry-api tests were failing to unmarshal this response. Sentry was returning back "{\"detail\": \"You must specify a list of IDs for this operation\"}"

**Review by mattrobenolt — APPROVED** [review]
Weird. Not sure how we did this. 😕

**Comment by mattrobenolt** [comment]
Thanks! 🍕  🍪

**Comment by cwood** [comment]
No worries. I thought it was an issue with go and then saw the code. Thanks for merging :)

**Pull request merged**